### ISSUE TITLE
RestWebhookMessageEditPayload Class Missing Attachments/Files Property

### ISSUE THREAD

**Issue by OoLunar** [DSharpPlus/DSharpPlus#942]
# Summary
According to <https://discord.com/developers/docs/resources/webhook#edit-webhook-message>, you can add/edit/remove attachments from a sent webhook message. The [RestWebhookPayload](https://github.com/DSharpPlus/DSharpPlus/blob/master/DSharpPlus/Net/Abstractions/Rest/RestWebhookPayloads.cs#L69-L82) doesn't implement this at all. 

# Details
Due to lack of the Attachments/Files property, any third party libraries that use webhook objects, such as @IDoEverything's [Slash Command](https://github.com/IDoEverything/DSharpPlus.SlashCommands) library, won't be able to add/edit/remove files. Once the webhook message is sent, I cannot [edit files](https://github.com/DSharpPlus/DSharpPlus/blob/master/DSharpPlus/Net/Rest/DiscordApiClient.cs#L2090-L2096) with the following code:
```cs
DiscordWebhookBuilder discordWebhookBuilder = new();
discordWebhookBuilder.Content = "No text found, turned the embeds into JSON!";

DiscordWebhook webhook = await context.Channel.CreateWebhookAsync("Do Webhooks Allow Files", default, "DSharpPlus Testing");
DiscordMessage webhookMessage = await webhook.ExecuteAsync(discordWebhookBuilder);
discordWebhookBuilder.AddFiles(messageFiles, true);
await webhook.EditMessageAsync(webhookMessage.Id, discordWebhookBuilder);
await webhook.DeleteAsync();
```

@Lulalaby had a fix available in their massive #890 PR, specifically in commit https://github.com/DSharpPlus/DSharpPlus/commit/8c24b9d9460705f355e2debbfb984e490322cde3.

**Comment by VelvetThePanda** [comment]
@IDoEverything p sure this should be marked as bug

**Comment by IDoEverything** [comment]
> @IDoEverything p sure this should be marked as bug

not really a bug, just an unimplemented feature

**Comment by IDoEverything** [comment]
fixed in #953
